src/TodoList.js

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import TodoItems from './TodoItems'
+import Footer from "./Footer";
+
+class TodoList extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            value: '',
+            items: [],
+            filter: 'all'
+        };
+
+
+    }
+
+    onChange = (event) => {
+        this.setState({value:event.target.value})
+    };
+
+
+    onSubmit = (event) => {
+        event.preventDefault();
+
+        let newItem = {
+            text: this.state.value,
+            key: Date.now(),
+            done: false,
+            visibility: null
```

Comment:
```suggestion
            visibility: false
```
It also can be boolean